packages/teleport/src/config.ts

```diff
@@ -111,6 +111,7 @@ const cfg = {
     applicationsPath:
       '/v1/webapi/sites/:clusterId/apps?searchAsRoles=:searchAsRoles?&limit=:limit?&startKey=:startKey?&query=:query?&search=:search?&sort=:sort?',
     clustersPath: '/v1/webapi/sites',
+    clusterAlertsPath: '/v1/webapi/sites/:clusterId/alerts',
```

Comment:
do we get cluster specific alerts? like if we change a cluster, can we get a different set of alerts?

Reply:
Technically the backend supports it so I felt it best to include it here and in the apiserver endpoint.